### PR TITLE
Properly Initialize Repo + Code Health

### PR DESCRIPTION
**Pull request by goobta**
Just doing what I should have done a while ago. Since I hadn't disable push access to `master` until today.... I've kinda just been pushing to it (read: **power trip**). So for documentation's case, I'll list down everything's that been done so far:
- Set up CI / CD (run tests + coverage)
- Created a test to ensure that jupyter notebooks and scripts are links at push time
- Created `auto_trainer`
  - So far only works with `stable_baselines` --will probably need to change that in the near future
  - Dynamic W&B compatibility
  - Docstrings
  - Tests
  - Turned into a package for auto install

So far, this is using `stable-baselines` as the algorithm implementations. If I can't figure out why `stable-baselines`'s PPO2 implementation is causing a memory leak, my next step is going to be to create a `Model` interface so I can implement my own algorithms using Tensorflow 2 and/or [TF-Agents](https://www.tensorflow.org/agents).

### CODE REVIEW

File: auto_trainer/trainer.py [hunk 1]
```diff
@@ -21,7 +25,22 @@
   _WANDB = False
 
 
-def get_synced_config(parameters, tags):
+def get_synced_config(parameters, tags: List[Text]):
+  """Sync the config with wandb (if necessary) and return the new config
+
+  Args:
+    parameters (Any W&B supported type): The current hyperparameters to use. If
+      W&B is enabled and is actively making making a sweep, these
```
Comment (mahajanrevant) [inline auto_trainer/trainer.py]: Remove one of the "making" in this sentence

Comment (goobta) [reply auto_trainer/trainer.py]: done

**Review by goobta — COMMENTED** [review]
Comments addressed

**Review by mahajanrevant — APPROVED** [review]
LGTM

**Pull request merged**